sql/src/test/java/io/crate/executor/transport/TransportShardUpsertActionTest.java

```diff
@@ -204,14 +204,14 @@ public void testExceptionWhileProcessingItemsContinueOnError() throws Exception
         ).newRequest(shardId, null);
         request.add(1, new ShardUpsertRequest.Item("1", null, new Object[]{1}, null));
 
-        TransportWriteAction.WriteResult<ShardResponse> result = transportShardUpsertAction.processRequestItems(
+        Tuple<ShardResponse, Translog.Location> shardResponseLocationTuple = transportShardUpsertAction.processRequestItems(
             shardId, request, new AtomicBoolean(false));
 
-        ShardResponse response = result.getResponse();
+        ShardResponse response = shardResponseLocationTuple.v1();
         assertThat(response.failures().size(), is(1));
         assertThat(response.failures().get(0).message(),
-                   is("VersionConflictEngineException[[default][1]: version conflict, " +
-                      "document with id: 1 already exists in 'characters']"));
+            is("VersionConflictEngineException[[default][1]: version conflict, " +
```

Comment:
seems not needed, pls don't apply such changes at story branches, will just make rebasing more complicated